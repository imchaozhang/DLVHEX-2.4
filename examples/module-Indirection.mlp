#module(p1,[]).
t.
p.
r:-t.
q:-r.
s:-not p.
ok :- @p2[q,s]::b.

#module(p2,[q/0,s/0]).
b:-q, not s.


