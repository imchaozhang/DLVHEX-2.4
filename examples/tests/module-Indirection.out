(p1[{}]={t,p,r,q,ok}, p2[{q}]={q,b})
